python/minetest/minetest_env.py

```diff
@@ -737,7 +758,13 @@ def read_config_file(file_path):
     return config
 
 
+def _value_to_config_string(value):
```

Comment:
Is this necessary?

Reply:
Not sure, but the [example conf uses lowercase](https://github.com/Astera-org/minetest/blob/d11359409aafb1111dc0f8058524512382dca458/minetest.conf.example#L41-L46) and I like consistency.